lib/helpers/results_helpers.js

```diff
@@ -257,32 +257,37 @@ module.exports = class ResultsHelpers {
             statementResults[lib][statementName].pretty = `FALSE (${rawStatementResult})`;
           }
         }
-        // create clause results for all localIds in this statement
-        const localIds = MeasureHelpers.findAllLocalIdsInStatementByName(measure, lib, statementName);
-        for (const localId in localIds) {
-          const clause = localIds[localId];
-          const clauseResult = {
-            // if this clause is an alias or a usage of alias it will get the raw result from the sourceLocalId.
-            raw:
-              rawClauseResults[lib] != null
-                ? rawClauseResults[lib][clause.sourceLocalId != null ? clause.sourceLocalId : localId]
-                : undefined,
-            statementName,
-          };
-          clauseResult.final = this.setFinalResults({
-            statementRelevance,
-            statementName,
-            rawClauseResults,
-            lib,
-            localId,
-            clause,
-            rawResult: clauseResult.raw,
-          });
-          clauseResults[lib][localId] = clauseResult;
+
+        if (includeClauseResults) {
```

Comment:
Need to add a test to assert that these are only included when `includeClauseResults` is set to true